jekyll/_cci2/[WIP] runner-overview.adoc

```diff
@@ -0,0 +1,96 @@
++
+** IP restrictions - Runners can have static IP addresses that you can control.
+** Identity Access Management (IAM) permissions - If you set up runners in AWS, they can be assigned IAM permissions.
+** Monitor the operating system
+** Connect to private networks
+
+* Unique compute requirements - Customers who need to run jobs on an environment or architecture that CircleCI does not offer as a resource class can use the runner to fill that need.
+
+== Runner Operation
+
+Once CircleCI Runner is installed, the runner polls circleci.com for work, runs jobs, and returns status, logs, and artifacts to CircleCI. When the runner is not running a job, it will auto-update itself when a new version is released.
+
+The runner consists of two components: the Launch Agent and the Task Agent
+
+* Launch Agent (launch-agent) - manages gathering the information required to run a task, while also downloading and launching a Task Agent process.
+* Task Agent (task-agent) - handles running a task retrieved and configured by the Launch Agent.
+
+The system has been written to allow agent administrators to configure the task-agent to run with a lower level of privileges than the launch-agent - as any user who is able to execute a job will be able to gain the same privileges as task-agent. The instructions below are for our recommended deployment which follows this approach - Launch Agent will run as root, but Task Agent will run as circleci.
+
+== CircleCI Runner Limitations
+
+There are a few limitations you should be aware of when using CircleCI Runner:
+
+The following platforms are currently supported:
+
+* Intel + Ubuntu
+* Arm64 + Ubuntu
+* Intel + macOS
+
+For more information on additional platforms that have been tested, but are not supported, and platforms that have not veen tested, please refer to the Runner Installation [add link here when ready] page.
+
+The following features are not yet available, but may be in the future:
+
+* Rerun with SSH
+* Test splitting
+* add_ssh_keys
+
+== Public repositories
+
+CircleCI Runner is not recommended for use with public projects that have the "Build forked pull requests" setting enabled. In this case, a malicious actor may alter your machine or execute code on it by forking your repository, committing code, and opening a pull request. Untrusted jobs running on your CircleCI Runner pose significant security risks for your machine and network environment, especially if your machine persists its environment between jobs. Some of the risks include:
+
+* Malicious programs running on the machine.
+* Escaping the machine's runner sandbox.
+* Exposing access to the machine's network environment.
+* Persisting unwanted or dangerous data on the machine.
+
+== Prerequisites
+
+In order to use CircleCI Runner, you must:
+
+* be on the Scale Plan.
+* use CircleCI Cloud.
+* have experience managing your infrastructure.
+* install [add link here when ready] CircleCI Runner.
```

Comment:
Capitalise initial words for consistency with other lists I think?

Reply:
Might be worth syncing up with Vivek and @appplemac about how we want to word this section

Reply:
Fixed this issue and also reached out to Alexey and Vivek for better messaging in this section.